src/Symfony/Component/Intl/Countries.php

```diff
@@ -52,6 +76,20 @@ public static function exists(string $country): bool
         }
     }
 
+    /**
+     * @param string $alpha3Code Alpha3 country code
+     */
```

Comment:
This phpdoc can be removed (we don't add them when they do not add any information already contained in the method signature).

Reply:
Done